miqa/core/models/image.py

```diff
@@ -23,13 +23,18 @@ class Meta:
     raw_path = models.CharField(max_length=500, blank=False, unique=True)
     name = models.CharField(max_length=255, blank=False)
 
+    def clean(self):
+        super().clean()
+        nifti_to_zarr_ngff.delay(self.path)
+
     @property
     def path(self) -> Path:
         return Path(self.raw_path)
 
     @property
     def zarr_path(self: Image) -> Path:
-        return nifti_to_zarr_ngff(self.path)
+        # return nifti_to_zarr_ngff(self.path)
```

Comment:
Is this comment a future TODO, or a forgotten leftover from the past?

Reply:
This was a note-to-self, but we can remove it.